src/components/one_click_shipping_address_wrapper/OneClickShippingAddressWrapper.jsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/forbid-prop-types */
+import React, { useCallback } from 'react';
+import ShippingAddress from '../shipping_address/ShippingAddress';
+import useShippingAddress from '../../hooks/useShippingAddress';
+
+const OneClickShippingAddressWrapper = () => {
+    const { submitShippingAddress } = useShippingAddress();
+
+    const trySubmitAddress =  useCallback( (localStateAddress) => {
```

Comment:
It doesn't look like this function is ever used and the above hook would not be used either. @jonathanzredfern

Reply:
Yea, this entire class isn't necessary anymore actually. I removed it in another task so when that gets merged it will be gone.